### PR TITLE
Logging adjustments

### PR DESCRIPTION
**Pull request by lava**
###  :notebook_with_decorative_cover: Description

1. Change the default log patterns so that day and time zone are not printed for the console logs, and the time zone *is* printed for file logs
2. Provide separate macros `VAST_TRACE()` and `VAST_TRACE_SCOPED()` to allow trace-level logging without creating a scope guard.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries. (not necessary, since the previous log pattern was never released)
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Review commit-by-commit. You can actually regard the commits as individual PRs.

### CODE REVIEW

File: libvast/src/chunk.cpp [hunk 1]
```diff
@@ -192,7 +192,7 @@ chunk::chunk(view_type view, deleter_type&& deleter) noexcept
   : view_{view}, deleter_{std::move(deleter)} {
   auto data = view.data();
   auto sz = view.size();
-  VAST_TRACEPOINT(chunk_make, data, sz);
+  VAST_TRACE_SCOPEPOINT(chunk_make, data, sz);
```
Comment (dominiklohmann) [inline libvast/src/chunk.cpp:195]: That does not look correct, your replacement sed should've included the opening bracket.

**Review by dominiklohmann — COMMENTED** [review]
Looks good, just fix up the VAST_TRACEPOINT macro before merging please.

Can you also change the vast.yaml.example accordingly?

File: libvast/src/chunk.cpp (hunk 1 above)
Comment (lava) [reply libvast/src/chunk.cpp]: Whoops, updated.

**Review by dominiklohmann — APPROVED** [review]
LGTM

**Pull request merged**